all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p><code>math.floor</code>  and <code>math.ceil</code>might be more intuitive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>